includes/admin/templates/settings/exclusions.php

```diff
@@ -84,11 +87,36 @@
                 <p class="description"><?php echo esc_html__('Set a threshold for daily robot visits. Robots exceeding this number daily will be identified as bots.', 'wp-statistics'); ?></p>
             </td>
         </tr>
+        <?php
+            $useHoneypot = ! empty(WP_STATISTICS\Option::get('use_honeypot'));
 
+            if ($useHoneypot) {
+                Notice::renderNotice( 
+                    sprintf(
+                        wp_kses(
+                            /* translators: %1$s: opening strong tag, %2$s: closing strong tag, %3$s: Learn more link */
+                            esc_html__('The WP Statistics %1$sHoney Pot Trap Page%2$s option will be removed in version 14.13. %3$s', 'wp-statistics'),
+                            [
+                                'strong' => [],
+                                'a' => [
+                                    'href' => [],
+                                    'target' => [],
+                                ],
+                            ]
+                        ),
+                        '<strong>',
+                        '</strong>',
+                        '<a href="https://wp-statistics.com/resources/deprecating-the-honey-pot-trap-page-option/?utm_source=wp-statistics&utm_medium=link&utm_campaign=settings" target="_blank">Learn more</a>'
```

Comment:
Let's make sure here is also translatable as well. @hoomanhlm

Reply:
@mostafasoufi I updated it, in both cases it was translatable, but now it's easier.